docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,19 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services. To acquire the template service source code the user needs to fetch the code from github.
```

Comment:
@rpannek the template has those files but I guess it makes sense to describe the structure here. To what extent though? Sometimes stating the obvious could be confusing, I would refrain from writing things like README.md is the readme file, etc.